### ISSUE TITLE
3.13.1 brings a fatal error in specific config

### ISSUE THREAD

**Issue by MeCapron** [PHP-CS-Fixer/PHP-CS-Fixer#6720]
## Bug report

The bug was introduced by this commit : https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/commit/b577444c38b61f2fdc32cd0a4685bd595be391b7

Since the update of the 3.13.1, a new error is shown : 

```
In FixCommand.php line 318:
                                                                              
  [ErrorException]                                                            
  count(): Parameter must be an array or an object that implements Countable  
                                                                              
Exception trace:
  at /builds/adexos/projects/curr/proj/vendor/friendsofphp/php-cs-fixer/src/Console/Command/FixCommand.php:318
 {closure}() at /builds/adexos/projects/curr/proj/vendor/friendsofphp/php-cs-fixer/src/Console/Command/FixCommand.php:318
```

It seems to be related with the finder which is not specifically countable.

As far as I can tell, the method `getFinder` from the resolver returns a n iterable. An iterable does mean a specifcy type that implements `Traversable` but not `Countable`

### Code snippet that reproduces the problem

You can use a specific finder with the path-mode to intersection to reproduce the problem, with the output as null.

And then you can reproduce the problem with this command :

```bash
vendor/bin/php-cs-fixer --allow-risky=yes --config=./.php-cs-fixer.php --path-mode=intersection --verbose --format=json fix /path/to/your-file.php --dry-run 
```

and this configuration

```php
<?php

declare(strict_types=1);

/**
 * Copyright © Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */

/**
 * PHP Coding Standards fixer configuration.
 */
$finder = PhpCsFixer\Finder::create()
    ->in('./')
    ->name(['*.php', '*.php']);

$config = new PhpCsFixer\Config();
$config->setFinder($finder);

return $config;
```

More specifically, if you set the verbosity and the format is to json, your `$progressType` will be null , so your $finder won't be 
converted to a countable.

https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/b577444c38b61f2fdc32cd0a4685bd595be391b7/src/Console/Command/FixCommand.php#L283

Then you will face the error at : https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/b577444c38b61f2fdc32cd0a4685bd595be391b7/src/Console/Command/FixCommand.php#L318

**Comment by jschaedl** [comment]
prabably same as #6718

**Comment by keradus** [comment]
indeed a duplicate

Fix released as [v3.13.2](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/releases/tag/v3.13.2)
